README.md

```diff
@@ -21,7 +21,7 @@ Create a Python3 Virtual Environmentm, activate it and install the development r
 ```
 virtualenv -p python3 venv
 source venv/bin/activate
-pip install -r requirements/local.txt
+pip3 install -r requirements/local.txt --user
```

Comment:
We don't want to install the requirements systemwide. Good for Containers, but these are general instructions.